setup.py

```diff
@@ -11,6 +11,7 @@
     "click<8.0",
     "semver~=2.10",
     "python-slugify>=4.0.1",
+    "tabulate==0.8.9",
```

Comment:
I wonder how the tests work without this dependency :D Good catch!

Reply:
Tests are passing because tabulate is a transitive dependency in test scope. Indeed good catch.